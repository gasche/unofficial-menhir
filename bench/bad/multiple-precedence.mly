%left A
%right A
%token A
%start<int> a
%%

a: A {}
