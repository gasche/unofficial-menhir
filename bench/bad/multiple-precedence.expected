File "multiple-precedence.mly", line 2, characters 7-8:
File "multiple-precedence.mly", line 1, characters 6-7:
Error: there are multiple precedence declarations for token A.
